site/content/contribute/mobile/developer-setup/_index.md

```diff
@@ -79,14 +79,22 @@ $ npm -g install react-native-cli
 #### Install Ruby
 ##### Windows 10
 - Install Ruby from [here](https://rubyinstaller.org/)
+
+##### MacOs
+
+While MacOs supply a ruby environment, it is being deprecated and it will only suffice if your architecture is intel based. For M1 it is recommended to install ruby via homebrew
```

Comment:
```suggestion
While MacOS supplies a Ruby environment, it's being deprecated and it will only suffice if your architecture is Intel-based. For M1-based architecture, it's recommended to install Ruby using Homebrew.
```